worlds/ExportWorld.py

```diff
@@ -0,0 +1,100 @@
+# TODO: handle data
+# TODO: handle clients
+# TODO: handle yaml templates
+import argparse
+import json
+import os.path
+import platform
+import sys
+import zipfile
+from pathlib import Path
+
+if __name__ == "__main__":
+    sys.path.remove(os.path.dirname(__file__))
+    ap_root = os.path.normpath(os.path.join(os.path.dirname(__file__), os.pardir))
+    os.chdir(ap_root)
+    sys.path.append(ap_root)
+
```

Comment:
I think it would be better to keep all the entry points in one place, just one of the multiple reasons being so we don't have to deal with `sys.path` hacks like this.

An `Archipelago/ExportWorld.py` with just something like this would be better.
```python
import argparse
from pathlib import Path

from worlds.AutoWorld import AutoWorldRegister
from worlds.export import export_world

if __name__ == "__main__":
    ap_root_path = Path(__file__).parent
    print(f"{ap_root_path=}")

    parser = argparse.ArgumentParser("simple_example")
    parser.add_argument("world_name", help="World to be exported.", type=str)
    parser.add_argument("output_dir", help="Export directory.", type=Path, default=ap_root_path / "worlds", nargs='?')
    args = parser.parse_args()

    output = export_world(ap_root_path, AutoWorldRegister.world_types[args.world_name], args.output_dir,
                          is_frozen=False)
    print(f"Output to {output}")
```

Reply:
I agree, I just figured it would be best to discuss first since that portion was not my code.